CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # ChangeLog
 
+## 3.2.4 - 2024-06-15
+
+Fixes:
+- fix bug in getRecordings endpoint
```

Comment:
please add a clear statement for security fixes and details about the attack vector